docs/_Research-Data-Management/26-licenses.md

```diff
@@ -51,6 +51,22 @@ There are different versions of CC that consist of the core license with further
 ## Usage
 Creators can declare a works license by selecting it during upload to a website/database/repository or attaching a line stating the title, author and license to the published work.
 
+Here we look at some examples of license usage, found on [Zenodo](https://zenodo.org/).
+
+### [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) example usage
```

Comment:
HAving hyperlinks in headings is usually frowned upon as it can collide with autogenerated permalink/anchors for the headings in TOC or cause accessibility issues in some browsers
```suggestion
### CC BY 4.0 example usage
```